Flow.Launcher.Core/Updater.cs

```diff
@@ -29,89 +29,69 @@ public Updater(string gitHubRepository)
             GitHubRepository = gitHubRepository;
         }
 
-        public async Task UpdateApp(IPublicAPI api , bool silentUpdate = true)
+        public async Task UpdateApp(IPublicAPI api, bool silentUpdate = true)
         {
-            UpdateManager updateManager;
-            UpdateInfo newUpdateInfo;
-
-            if (!silentUpdate)
-                api.ShowMsg("Please wait...", "Checking for new update");
-
             try
             {
-                updateManager = await GitHubUpdateManager(GitHubRepository);
-            }
-            catch (Exception e) when (e is HttpRequestException || e is WebException || e is SocketException)
-            {
-                Log.Exception($"|Updater.UpdateApp|Please check your connection and proxy settings to api.github.com.", e);
-                return;
-            }
+                UpdateInfo newUpdateInfo;
+
+                if (!silentUpdate)
+                    api.ShowMsg("Please wait...", "Checking for new update");
+
+                using var updateManager = await GitHubUpdateManager(GitHubRepository);
```

Comment:
back when i was working on this section of the code i read Squirrel had some reported issues where it is not properly disposing the updateManager, could we double check this issue is no longer relevant in our Squirrel version please. I think it was to do with certain Squirrel versions and when this project was still on .Net Framework

Reply:
I remember there is a comment on this part of code that said we need to manually dispose the updatemanager for squirrel 1.5.2, but the use of using will be fine since it is equal to dispose at the end of the scope.